lib/mspec/helpers/tmp.rb

```diff
@@ -41,6 +41,7 @@ def tmp(name, uniquify = true)
   if uniquify and !name.empty?
     slash = name.rindex "/"
     index = slash ? slash + 1 : 0
+    name = +name
```

Comment:
I think `.dup` might be nicer here, because then this method simply never mutates the argument, which feels cleaner, could you switch to that?